### DISCUSSION TITLE
Автофокус при закрытии селекта через tuiTextfieldCleaner

### DISCUSSION THREAD

**Discussion by zavod008** [taiga-family/taiga-ui#5597] [Q&A] [answered]
Привет, заранее спасибо.

В текущей реализации при закрытии комбобокса через крест, он сразу делает фокус в инпут и появляются варианты выбора.
Можно ли без подписки на евенты убрать это поведение?
Чтобы когда я нажимаю на крест значение просто сбрасывалось и ничего больше не происходило.

**Comment by waterplea** [comment, marked as answer]
Заведи, пожалуйста, баг — в селекте этот момент предусмотрен, а в комбобоксе и мультиселекте/инпуттэге нет.
